images/macos/scripts/docs-gen/Generate-SoftwareReport.ps1

```diff
@@ -311,7 +311,13 @@ if ($os.IsBigSur -or $os.IsMonterey) {
     $miscellaneous.AddToolVersion("Zlib", $(Get-ZlibVersion))
 }
 
-if ($os.IsMonterey) {
+if ($os.IsSonoma -or $os.IsVentura) {
```

Comment:
Should this block also have `IsMonterey`?

Reply:
For `IsMonterey' we have a separate block above it with additional items required for it.

```
if ($os.IsBigSur -or $os.IsMonterey) {
    $miscellaneous = $installedSoftware.AddHeader("Miscellaneous")
    $miscellaneous.AddToolVersion("libXext", $(Get-LibXextVersion))
    $miscellaneous.AddToolVersion("libXft", $(Get-LibXftVersion))
    $miscellaneous.AddToolVersion("Tcl/Tk", $(Get-TclTkVersion))
    $miscellaneous.AddToolVersion("Zlib", $(Get-ZlibVersion))
}
```

Reply:
I see that now 👍